utils/token_tracker.py

```
@@ -71,32 +76,164 @@ def _deep_copy_day(day: dict) -> dict:
             t_bt[ct][k] = t_bt[ct].get(k, 0) + bucket.get(k, 0)
 
 
+# ---------------------------------------------------------------------------
+# 跨进程文件锁（O_CREAT | O_EXCL 方式，跨平台）
+# ---------------------------------------------------------------------------
+
+@contextmanager
+def _file_lock(lock_path: Path, timeout: float = 10.0):
+    """基于文件系统的跨进程互斥锁。
+
+    使用 O_CREAT | O_EXCL 原子创建锁文件，确保同一时刻只有一个进程持有锁。
```

Ruff (0.15.5)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)